### PR TITLE
Add Deepnest.app v1.0.5

### PR DESCRIPTION
**Pull request by DanLipsitt**
After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/deepnest.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+cask "deepnest" do
+  version "1.0.5"
+  sha256 "602c97f2becfddd808bfea8cab1cd7c20503494fa17df2ee55b3cec65a45c7eb"
+
+  url "https://deepnest.io/Deepnest-#{version}-mac.zip"
+  name "deepnest"
+  desc "/opt/homebrew/Library/Taps/homebrew/homebrew-cask/Casks/deepnest.rb`"
```
Comment (SMillerDev) [inline Casks/deepnest.rb:7]: https://docs.brew.sh/Cask-Cookbook#stanza-desc

File: Casks/deepnest.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  homepage "https://deepnest.io/"
+
+  livecheck do
+    url :homepage
+    strategy :page_match
```
Comment (SMillerDev) [inline Casks/deepnest.rb:12]: This is the default, I don't think it's needed.

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @DanLipsitt.

Could you please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap), thanks.

**Comment by miccal** [comment]
Closing due to lack of reply.

**Pull request closed without merging**